src/cascadia/TerminalSettingsModel/ActionArgs.cpp

```diff
@@ -814,10 +814,25 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
 
     winrt::hstring SearchForTextArgs::GenerateName() const
     {
-        return winrt::hstring{
-            fmt::format(std::wstring_view(RS_(L"SearchForTextCommandKey")),
-                        Windows::Foundation::Uri(QueryUrl()).Domain().c_str())
-        };
+        if (QueryUrl().empty())

```

Comment:
I'm confused… Isn't this supposed to be `if (QueryUrl())`?
(If it is: You can use `if (const auto queryUrl = QueryUrl())` here.)

Reply:
nah cause like, the first branch is for "the user didn't provide a `queryUrl`, so let's just display this action with the default name".